goodmap/data_validator.py

```diff
@@ -0,0 +1,65 @@
+import json
+from sys import argv, stderr
+
+
+def are_obligatory_fields_present(datapoints, obligatory_fields):
```

Comment:
If function starts with `are` or `is` it should always return bool.

Reply:
Probably better function name would be `get_missing_obligatory_fields` which would return list of missing obligatory fields. And different function would print it.